test/integration/MetaDataAtom.spec.ts

```diff
@@ -50,15 +51,12 @@ describe('RLAU-572: MetaData in Atoms', () => {
 
         identity1 = identityManager.generateSimpleIdentity()
         identity2 = identityManager.generateSimpleIdentity()
-
-        await identity1.account.openNodeConnection()
-        await identity2.account.openNodeConnection()
+        
         nodeConnection = await radixUniverse.getNodeConnection(identity1.address.getShard())
     })
 
     after(async () => {
-        await identity1.account.closeNodeConnection()
-        await identity2.account.closeNodeConnection()
+        //
```

Comment:
Is this some "prevent-auto-format-to-do-what-it-is-configured-to-do" magic?

Reply:
It's just me being lazy, I should take out the whole function. But also, I refuse to auto-format :D

Reply:
A man of culture! 😆